.pre-commit-config.yaml

```diff
@@ -40,9 +40,8 @@ repos:
         rev: v11.1.0
         hooks:
               - id: clang-format
-                files: \.(cu|cuh|h|hpp|cpp|inl)$
-                types_or: [file]
-                args: ['-fallback-style=none', '-style=file', '-i']
+                types_or: [c, c++, cuda]
```

Comment:
This is identical to the previous behavior according to the [identify extensions definition](https://github.com/pre-commit/identify/blob/main/identify/extensions.py).

Reply:
We still have one `.inl` file (strings/regex/regex.inl`) which I'm not sure is covered by this change. Did we pin a specific `identity` version to make sure this works?

Reply:
We might not _need_ to pin `identify` because your changes have been released for a while and the CI images running this are updated pretty often. We can add that pinning to https://github.com/rapidsai/integration/blob/branch-22.10/conda/recipes/rapids-build-env/meta.yaml but it might not be necessary.

Reply:
Sounds good to me.